webplatformcompat/view_serializers.py

```diff
@@ -21,7 +21,14 @@
 from .serializers import (
     BrowserSerializer, FieldMapMixin, FeatureSerializer, MaturitySerializer,
     SectionSerializer, SpecificationSerializer, SupportSerializer,
-    VersionSerializer, omit_some)
+    VersionSerializer, FieldsExtraMixin)
+
+
+def omit_some(source_list, *omitted):
+    """Return a list with some items omitted"""
+    for item in omitted:
+        assert item in source_list, '%r not in %r' % (item, source_list)
```

Comment:
How big do the `source_list` and `omitted` lists tend to be? Also, how often does `omit_some` get called?

While I recognize that this assert makes sure things are correct, this seems expensive if the numbers of things are "high" for some threshold of "high".